pkg/scheduler/metrics/job.go

```diff
@@ -57,4 +83,6 @@ func DeleteJobMetrics(jobName, queue, namespace string) {
 	unscheduleTaskCount.DeleteLabelValues(jobName)
 	jobShare.DeleteLabelValues(namespace, jobName)
 	jobRetryCount.DeleteLabelValues(jobName)
+	jobCompletedPhaseCount.DeleteLabelValues(namespace, jobName)
```

Comment:
The new added metrics exist in vc controller but these are defined in scheduler, this is a liitel wired: )

Reply:
+1, move to pkg/controller is better

Reply:
ok